target_position_source_frame: base_link # origin frame for position of the target (In our case the source frame of our cube)
tf_source_frame: base_link # We are going to move the target by reference to this frame
tf_target_frame: camera_link # target frame are going to calibrate
target_position: # position of the target in the target_position_source_frame
    x: 0.5
    y: 0.0
    z: 0.05
target_rotation:
    roll: 0
    pitch: 0
    yaw: 0
estimated_sensor_position:
    x: 0
    y: 0
    z: 0
estimated_sensor_rotation:
    roll: 0
    pitch: 0
    yaw: 0
cube_scale:
    x: 1
    y: 1
    z: 1
cube_alpha: 0.5 # A value of zero would make the cube completely transparent
